client/scss/tools/_functions.general.scss

```diff
@@ -0,0 +1,15 @@
+@use 'sass:math';
+
+@function strip-unit($value) {
+  @return math.div($value, ($value * 0 + 1));
+}
+
+// Returns a value in rem
+@function rem($val, $context: 16) {
```

Comment:
My concern with this kind of function is it decouples the base REM units in px from those defined in the actual root CSS. Making it harder to customise. 

Also - creates a disconnect between what 1rem may be in the tailwind config. 

Can this context pull from something like tailwind? Or somehow be centralised.

Reply:
For now I’ve refactored this to remove the `em` function entirely, and have this `rem` function use a hard-coded context of 16px. This should make the function harder to mis-use, and if we want to take it away entirely it’ll be a simpler job.

I still think there is a case for this, though we could also directly define values in `rem` and just have the pixel value as a comment next if we preferred. Let me know what you think!

Reply:
Yeah. That's a good point - however - the base REM of 16 px could still be a bit of a friction point for syncing. 

Not a blocker for this PR though.

Reply:
discussed this further - and maybe it is better we try to use rems only and add a manual comment where we need that helps future devs know what the px value would be.

Reply:
I’ve removed this as discussed, and wrote all units in rem with the px values shown in code comments where relevant.